crates/egui-winit/src/lib.rs

```diff
@@ -549,6 +549,24 @@ impl State {
     }
 
     fn on_mouse_wheel(&mut self, delta: winit::event::MouseScrollDelta) {
+        let mut push_raw_event = || {
+            let (unit, delta) = match delta {
+                winit::event::MouseScrollDelta::LineDelta(x, y) => {
+                    (egui::MouseWheelUnit::Line, egui::vec2(x, y))
+                }
+                winit::event::MouseScrollDelta::PixelDelta(winit::dpi::PhysicalPosition {
+                    x,
+                    y,
+                }) => (egui::MouseWheelUnit::Pixel, egui::vec2(x as f32, y as f32)),
```

Comment:
Let's use points for this instead to be consistent across different DPI:s:
```suggestion
                }) => (egui::MouseWheelUnit::Pixel, egui::vec2(x as f32, y as f32) / self.pixels_per_point()),
```